src/UI/Minimap.re

```diff
@@ -178,74 +189,71 @@ let%component make =
               c => c.editorMinimapShowSlider,
               state.configuration,
             )) {
-          /* Draw current view */
+          /* Draw slider/viewport */
           Shapes.drawRect(
             ~transform,
             ~x=0.,
             ~y=
               rowHeight
-              *. float_of_int(Editor.getTopVisibleLine(editor, metrics) - 1)
+              *. float(Editor.getTopVisibleLine(editor, metrics) - 1)
               -. scrollY,
-            ~height=
-              rowHeight *. float_of_int(getMinimapSize(editor, metrics)),
-            ~width=float_of_int(width),
+            ~height=rowHeight *. float(getMinimapSize(editor, metrics)),
+            ~width=float(width),
             ~color=state.theme.scrollbarSliderHoverBackground,
             (),
           );
         };
 
         let cursorPosition = Editor.getPrimaryCursor(editor);
+
         /* Draw cursor line */
         Shapes.drawRect(
           ~transform,
-          ~x=0.,
+          ~x=Constants.leftMargin,
```

Comment:
Here I've also added a margin, as that makes the minimap stand out a bit more from the editor surface.